### ISSUE TITLE
v2.3.4 gives a error

### ISSUE THREAD

**Issue by poudenes** [gcobb321/icloud3#127]
```
2021-02-23 16:12:30 ERROR (SyncWorker_2) [custom_components.icloud3.device_tracker] 'isOld'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 2761, in _update_device_icloud
    location_isold_attr = location_data[ATTR_ISOLD]
KeyError: 'isOld'
2021-02-23 16:12:30 ERROR (SyncWorker_2) [custom_components.icloud3.device_tracker] âºINTERNAL ERROR-RETRYING (_update_device_icloud:OverallUpdate-'isOld')
2021-02-23 16:59:15 ERROR (SyncWorker_37) [custom_components.icloud3.device_tracker] 'isOld'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 2761, in _update_device_icloud
    location_isold_attr = location_data[ATTR_ISOLD]
KeyError: 'isOld'
2021-02-23 16:59:15 ERROR (SyncWorker_37) [custom_components.icloud3.device_tracker] âºINTERNAL ERROR-RETRYING (_update_device_icloud:OverallUpdate-'isOld')
2021-02-23 17:23:10 ERROR (SyncWorker_27) [custom_components.icloud3.device_tracker] 'isOld'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 2761, in _update_device_icloud
    location_isold_attr = location_data[ATTR_ISOLD]
KeyError: 'isOld'
2021-02-23 17:23:10 ERROR (SyncWorker_27) [custom_components.icloud3.device_tracker] âºINTERNAL ERROR-RETRYING (_update_device_icloud:OverallUpdate-'isOld')
2021-02-23 17:50:00 ERROR (SyncWorker_32) [custom_components.icloud3.device_tracker] 'isOld'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 2761, in _update_device_icloud
    location_isold_attr = location_data[ATTR_ISOLD]
KeyError: 'isOld'
2021-02-23 17:50:00 ERROR (SyncWorker_32) [custom_components.icloud3.device_tracker] âºINTERNAL ERROR-RETRYING (_update_device_icloud:OverallUpdate-'isOld')
2021-02-23 17:50:15 ERROR (SyncWorker_17) [custom_components.icloud3.device_tracker] 'isOld'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 2761, in _update_device_icloud
    location_isold_attr = location_data[ATTR_ISOLD]
KeyError: 'isOld'
2021-02-23 17:50:15 ERROR (SyncWorker_17) [custom_components.icloud3.device_tracker] âºINTERNAL ERROR-RETRYING (_update_device_icloud:OverallUpdate-'isOld')
```

**Comment by gcobb321** [comment]
It’s a known issue where some unused code is creating a problem. Sometimes restarting ic3 fixes it for a while, sometimes restarting HA does. You can Edit device_tracker.py, delete line 2761 that starts with `location_isold...`

**Comment by gcobb321** [comment]
v2.3.5 is available for testing in the GitHub development v2.3.5 directory
Download the zip file here
Unzip it into the icloud3 directory
Restart HA
It fixes the following bugs:

The location_isold error when the data is not available. This was obsolete and unused code that has been removed.
Fixed the problem with the zone’s friendly name created in the v2.3.4 update for zones with multiple words.
